plugin-server/src/main/ingestion-queues/session-recording-v2/teams/team-service.ts

```diff
@@ -52,9 +48,12 @@ export async function fetchTeamTokensWithRecordings(client: PostgresRouter): Pro
 
     return selectResult.rows.reduce(
         (acc, row) => {
-            acc[row.api_token] = { teamId: row.id, consoleLogIngestionEnabled: row.capture_console_log_opt_in }
+            acc[row.api_token] = {
+                teamId: row.id,
+                consoleLogIngestionEnabled: row.capture_console_log_opt_in,
+            }
             return acc
         },
-        {} as Record<string, TeamIDWithConfig>
+        {} as Record<string, TeamForReplay>
```

Comment:
interestingly there's still a duplicate TeamIDWithConfig here https://github.com/PostHog/posthog/blob/90175d065868ca42aa6471bc2b4356ac415c986e/plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts#L121 but we _should_ be deleting that whole folder soon so we can probably 🙈